spec/models/organization_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Organization do
+	it "1:  without http & https" do
+		org = Organization.new
+		org.web = "www.casa.com"
+		org.ensure_url
+		expect(org.web).to eq "http://www.casa.com"
+	end
+	it "2: with http" do
+		org = Organization.new
+  		org.web = "http://www.casa.com"
+  		org.ensure_url
+  		expect(org.web).to eq "http://www.casa.com"
+  	end
```

Comment:
`end` at 15, 3 is not aligned with `it "2: with http" do` at 10, 1<br>Tab detected.